### DISCUSSION TITLE
RSS feed timeout

### DISCUSSION THREAD

**Discussion by johafor** [Novik/ruTorrent#2661] [Q&A]
I have an RSS feed that I suspect can be slow to load and therefore times out every now and then. Is there a setting I can change to have it wait a couple of seconds more? 

The error: 

> Error loading feed. - [RSS-Connection-Error] Error: cURL could not retrieve the document, error 28.

TIA

**Comment by stickz** [comment]
A simple google search of `curl error 28` will explain more information. This is not related to ruTorrent.

**Reply by johafor** [reply]
Thanks for the reply. I am not super versed in these kinds of issues, so I had to start somewhere.
Would this be a client side or server side issue?

**Reply by stickz** [reply]
@johafor This is a server side issue. It most likely related to PHP or DNS. The first thing I would do is check is `php.ini`. Ensure the memory limit is set to 256M. Also, ensure the time limit is set to 12000 to allow 12 seconds. 

Also, ensure you're running `ruTorrent v4.3.1` as you may be missing server side dependences required for RSS feeds. The plugin will still function without these dependences. However, `v4.3.1` will give you a warning to help resolve the problem if applicable.
